### DISCUSSION TITLE
Fariz Dwi Pratama | fari

### DISCUSSION THREAD

**Discussion by farizdp** [onlyphantom/llm-python#24] [Practicum: Generative AI]
OLD : https://colab.research.google.com/drive/1cSDzPRMg6XpoeK920e1-ZSxQCmUlCuWD?usp=sharing

NEW : https://colab.research.google.com/drive/1CdSM4CcSvtFyz6fhiffxHFb643ERXXKt?usp=sharing

**Comment by onlyphantom** [comment]
Need the notebook to be a public link so I can access it for grading ! :)

**Reply by farizdp** [reply]
Done updated the link

**Comment by onlyphantom** [comment]
Hey Fariz, I don't think your model managed to get the answers for your queries! :) In both of the following cases, it didn't produce a satisfactory answer! 


```
queries = [
    "What are the top 5 companies by transaction volume on the first of this month?",
    "What are the most traded stock yesterday?",
    "What are the top 7 most traded stocks between 6th June to 10th June this year?"
]

> Entering new AgentExecutor chain...
To answer this query, I need to know the exact date of the first of this month. Could you please provide the date in the format YYYY-MM-DD?

> Finished chain.
Answer: 
 To answer this query, I need to know the exact date of the first of this month. Could you please provide the date in the format YYYY-MM-DD? 
======


Question: What are the most traded stock yesterday?

> Entering new AgentExecutor chain...
To find the most traded stocks yesterday, I need to know the exact date for yesterday. Could you provide me with the date in the format YYYY-MM-DD?

> Finished chain.
Answer: 
 To find the most traded stocks yesterday, I need to know the exact date for yesterday. Could you provide me with the date in the format YYYY-MM-DD? 
======
```

If you have time, I urge you to have a read at [the materials](https://docs.sectors.app/recipes/generative-ai-python/02-tool-use) or Zoom recording again to see if you could give those suggestions in the materials a try?

Good luck!

**Reply by farizdp** [reply]
Sorry wrong link, that old file
This for last updated

https://colab.research.google.com/drive/1CdSM4CcSvtFyz6fhiffxHFb643ERXXKt?usp=sharing
